tests/centralized_telescope.rs

```diff
@@ -15,12 +15,12 @@ const DATA_LENGTH: usize = 32;
 fn test(created_with_params: bool) {
     let mut rng = ChaCha20Rng::from_seed(Default::default());
     let nb_tests = 1_000;
-    let set_size = 1_000;
+    let set_size: u64 = 1_000;
     let params = Params {
         soundness_param: 10.0,
         completeness_param: 10.0,
-        set_size: 80 * set_size / 100,
-        lower_bound: 20 * set_size / 100,
+        set_size: set_size.saturating_mul(80).div_ceil(100),
```

Comment:
Does this trigger for you when you run clippy? I doesn't on my laptop or in CI, even though it should...

Reply:
It did trigger on the CI in the old PR